test_runner/fixtures/pageserver/allowed_errors.py

```diff
@@ -126,6 +126,8 @@ def scan_pageserver_log_for_errors(
     ".*startup_reconcile: Could not scan node.*",
     # Tests run in dev mode
     ".*Starting in dev mode.*",
+    ".*Starting in dev mode - authentication security checks are disabled.*",
```

Comment:
You don't need the first of these -- you only need to match the actual string that is added in this PR